ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionBuilderModule.java

```diff
@@ -404,26 +405,28 @@ private void logReceivedBuilderExecutionPayload(final ExecutionPayload execution
   private void logReceivedBuilderBid(final BuilderBid builderBid) {
     final ExecutionPayloadHeader payloadHeader = builderBid.getExecutionPayloadHeader();
     LOG.info(
-        "Received Builder Bid (Block Number = {}, Block Hash = {}, MEV Reward (wei) = {}, Gas Limit = {}, Gas Used = {})",
+        "Received Builder Bid (Block Number = {}, Block Hash = {}, MEV Reward (ETH) = {}, Gas Limit = {}, Gas Used = {})",
         payloadHeader.getBlockNumber(),
         payloadHeader.getBlockHash(),
-        builderBid.getValue().toDecimalString(),
+        weiToEth(builderBid.getValue()),
```

Comment:
We need to be careful when converting BigDecimal to strings. Depending on how small the value is, it might stringify it using scientific notation...

Example:
```
BigDecimal number = new BigDecimal("0.0000001");

System.out.println(number); 
// prints 1E-7

System.out.println(number.toPlainString());
// prints 0.0000001
```

I suggest we use `toPlainString()` when logging BigDecimals to be consistent.